ircbot/plugins/links.py

```diff
@@ -22,6 +22,14 @@ def allowed_to_process(bot, channel):
     return True
 
 
+def truncate_title(title):
+    title = title.replace("\n", "")
+    if len(title) > 120:
```

Comment:
Please move this out to a config file value, probably `config['IRC']['link_max_len']`

Reply:
Ok, just saw commit db3602a - I prefer that config file location, nice one :+1: